### PR TITLE
PPD 874 - add bca virtual account

### PR DESCRIPTION
**Pull request by azioarv**
Add BCA Virtual Account to API wrapper

`bca_va_charge = midtrans. bca_virtual_account_charge.post(charge_params)`

### CODE REVIEW

**Review by philiplambok — APPROVED** [review]
LGTM <3

File: README.md [hunk 1]
```diff
@@ -125,6 +125,55 @@ gopay_charge = midtrans.gopay_charge.post(charge_params)
...
 
+#### BCA Virtual Account Charge
+```ruby
+charge_params = {
+  "payment_type": "bank_transfer",
+  "transaction_details": {
+      "order_id": "order-with-bca-virtua-account",
+      "gross_amount": 12500
+  },
+  "item_details": [
```
Comment (rslhdyt) [inline README.md:136]: just curious, why do we need to define the item details when creating the VA number?

**Review by hendrixsilaen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (azioarv) [reply README.md:136]: base on midtrans API documentation ([https://api-docs.midtrans.com/?php#bca-virtual-account](https://api-docs.midtrans.com/?php#bca-virtual-account)) that's marked as a mandatory field with price, quantity, and name as a mandatory field for each item

Comment (rslhdyt) [reply README.md:136]: roger that

**Review by rslhdyt — APPROVED** [review]

**Pull request merged**